code/modules/mob/living/simple_animal/morph/morph_event.dm

```diff
@@ -0,0 +1,45 @@
+/datum/event/spawn_morph
+	var/key_of_morph
+
+/datum/event/spawn_morph/proc/get_morph(end_if_fail = 0)
+	key_of_morph = null
+	if(!key_of_morph)
+		var/list/candidates = get_candidates(ROLE_ALIEN)
```

Comment:
Sure, I gave us enough antag tags to hand out like candy, why not hand them out like candy